src/components/Task/Drawer/Form/Form.test.tsx

```diff
@@ -54,14 +70,13 @@ describe('TaskDrawerForm', () => {
           >
             <TaskDrawerForm
               accountListId="abc"
+              filter={mockFilter}
+              rowsPerPage={100}
               onClose={onClose}
               task={{
                 activityType: null,
                 contacts: {
                   nodes: [],
-                  pageInfo: { hasNextPage: false, hasPreviousPage: false },
-                  totalCount: 0,
-                  totalPageCount: 0,
```

Comment:
@OzzieOrca, I'm not quite sure why typescript was yelling at me to remove this. I stashed all my changes and the error went away, so it has to be something I added. Just not sure what.

Reply:
I assume this is affected by the GraphQL query and the codegen for it. Maybe these fields aren't requested.